larky/src/main/java/com/verygood/security/larky/modules/vgs/vault/DefaultVault.java

```diff
@@ -0,0 +1,184 @@
+package com.verygood.security.larky.modules.vgs.vault;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+import com.verygood.security.larky.modules.vgs.vault.spi.LarkyVault;
+import net.starlark.java.eval.EvalException;
+import net.starlark.java.eval.NoneType;
+import net.starlark.java.eval.Starlark;
+
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.regex.Pattern;
+
+
+public class DefaultVault implements LarkyVault {
+
+    private final Map<String, Object> persistentVaultStorage = new HashMap<>();
+    private final Map<String, Object> volatileVaultStorage = new HashMap<>();
+
+    private final Map<String, Map<String, Object>> storageConfig = new HashMap<String, Map<String, Object>>() {{
+        put("persistent", persistentVaultStorage);
+        put("volatile", volatileVaultStorage);
+    }};
```

Comment:
Let's have consistency between static collection initialization.
There are laready guava `ImmutableSet` init in `ModuleSupplier`
Let's have `ImmutableMap`